### DISCUSSION TITLE
Move project from repo to a different repo or up to the org level.

### DISCUSSION THREAD

**Discussion by matthewpersico** [community/community#7604] [Projects and Issues]
As stated. The project I was working on grew and moved to encompass more than one repo. I have to manually put the cross-repo PRs into issues. Please consider this ASAP for the Enterprise version.

**Comment by ericelliott** [comment]
Not just enterprise, please.

**Comment by s3638844** [comment]
or even propose a workaround, is there an API request that can export a project as CSV/XML/JSON and then reimport the output to the org

**Comment by Luna-Klatzer** [comment]
I'd like to add that it would be also great to have the ability to move account-based projects to a repository or organisation

**Comment by noahehall** [comment]
+1

**Comment by GMNGeoffrey** [comment]
A similar request: we'd like to move a project from one organization to another. It looks like this will be especially tricky because projects at the org level can only contain issues from that org, so if we move any of our repos... what happens? I'd also like to second the above comment that an import/export feature would go a long way to work around this issue, while probably being not as hard to support

**Reply by AtomicSponge** [reply]
+1 for an export/import too, that alone would work for me

**Reply by GMNGeoffrey** [reply]
Looks like this is captured better in https://github.com/github-community/community/discussions/12908. Beta projects are *only* at the org level, so transferring them to a repository would actually require mapping between the two constructs, whereas moving between orgs seems like it should be very straightforward.

**Reply by GMNGeoffrey** [reply]
I posted in that other discussion instead, but I threw together a hacky script for migrating my team's project. See https://github.com/github-community/community/discussions/12908#discussioncomment-3101869

**Comment by AtomicSponge** [comment]
+1
Created an org account and migrated some repos.  Would be handy to move the project as well.  Or even the ability to link projects from members of that org.

**Comment by GMNGeoffrey** [comment]
I had the related goal of migrating a new beta project between organizations and threw together a hacky script to do so for my team's project. You might find it useful as a starting point for at least uploading to a new org-level V2 Project. See https://github.com/github-community/community/discussions/12908#discussioncomment-3101869

**Comment by ThePieMonster** [comment]
Also looking to move projects (beta) from a personally account to a organizational account.

**Comment by kfklein15** [comment]
So when is this happening? :)

**Comment by dyanakiev** [comment]
up
